src/main/resources/application.yaml

```diff
@@ -64,3 +64,6 @@ tasks:
   mark-letters-posted: ${FTP_REPORTS_CRON:0 30 0 * * *}
   stale-letters-report: ${STALE_LETTERS_REPORT_CRON:0 30 1 * * *}
 
+encryption:
+  enabled: ${ENCRYPTION_ENABLED:false}
```

Comment:
update terraform

Reply:
Yes will be doing that as part of next PR when we integrate with service